scripts/build-steamlink-app.sh

```
@@ -49,7 +49,7 @@ mkdir -p $DEPLOY_FOLDER/steamlink/apps/moonlight/bin
 cp $BUILD_FOLDER/app/moonlight $DEPLOY_FOLDER/steamlink/apps/moonlight/bin/ || fail "Binary copy failed!"
 cp $SOURCE_ROOT/app/deploy/steamlink/* $DEPLOY_FOLDER/steamlink/apps/moonlight/ || fail "Metadata copy failed!"
 pushd $DEPLOY_FOLDER
-zip -r $INSTALLER_FOLDER/Moonlight-SteamLink-$VERSION.zip . || fail "Zip failed!"
+zip -r $INSTALLER_FOLDER/Moonlight-VPlus-SteamLink-$VERSION.zip . || fail "Zip failed!"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)